vit-servicing-station-lib/src/v0/endpoints/snapshot/routes.rs

```diff
@@ -25,10 +24,9 @@ pub fn filter(
 }
 
 pub fn update_filter(
-    context: UpdateHandle,
+    context: SharedContext,
```

Comment:
why did you change this? not sure what the purpose of this crate `v0::context::SharedContext` is?

Reply:
I have replaced and removed `sled` database usage with the diesel. `UpdateHandle` stored the context of the `sled` database, `v0::context::SharedContext` stores diesel database context, so that is why I have replaced it